pennylane/qchem/convert_openfermion.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the functions needed to convert OpenFermion ``QubitOperator`` objects to PennyLane ``Sum`` and ``LinearCombination`` and vice versa.
+"""
+
+import pennylane as qml
+from pennylane import numpy as np
+from pennylane.qchem.convert import _openfermion_to_pennylane, _pennylane_to_openfermion
+
+
+def from_openfermion(of_qubit_operator, wires=None, tol=None):
```

Comment:
Please update this function to optionally return a [Sum](https://docs.pennylane.ai/en/stable/code/api/pennylane.ops.op_math.Sum.html) object.